src/System.Management.Automation/engine/MshObject.cs

```diff
@@ -2457,7 +2457,7 @@ internal enum WriteStreamType : byte
     /// Serves as a placeholder BaseObject when PSObject's
     /// constructor with no parameters is used.
     /// </summary>
-    public class PSCustomObject
+    public sealed class PSCustomObject
```

Comment:
Causes test failure due to public API change:

https://dev.azure.com/powershell/PowerShell/_build/results?buildId=45285&view=ms.vss-test-web.build-test-results-tab&runId=1310450&resultId=100569&paneView=debug

Reply:
reverted